ledger-service/http-json/src/main/scala/com/digitalasset/http/Endpoints.scala

```diff
@@ -67,12 +68,23 @@ class Endpoints(
   import util.ErrorOps._
   import Uri.Path._
 
+  // Inspired by
```

Comment:
I assume the idea is to switch over to that once we use the dsl?

Reply:
Correct